test/github-package.test.js

```diff
@@ -608,6 +608,8 @@ describe('GithubPackage', function() {
     let workdirPath2, atomGitRepository2, repository2;
 
     beforeEach(async function() {
+      this.retries(5); // FLAKE
```

Comment:
@smashwilson do we actually want this in the `beforeEach`? Is the goal with this to retry all of the tests in this `describe` block?

Reply:
My intent was to retry the `beforeEach` itself if it throws. I was going off of #761 which puts the failure there instead of within any of the specific tests.

I'm... not actually sure if that's what Mocha will do here, though? The [documentation](https://mochajs.org/#retry-tests) doesn't make it clear. It doesn't break though?